apps/events/models.py

```diff
@@ -496,14 +497,14 @@ def published_action(self):
             trigger = self.action.static_trigger
             trigger = StaticTrigger.objects.get_published_version(trigger)
             return trigger.action
-        except StaticTrigger.DoesNotExist:
+        except ObjectDoesNotExist:
```

Comment:
Interesting. Why this change?

Reply:
I forget why I changed that but I think I'll change it back